### PR TITLE
2339 Add New Tag Types

### LINKED ISSUE #2339: Maybe have an uplift or height difference tag?
In response to this screenshot:
image

We may want to have an "uplift" or "height difference" tag for surface problem? See this [categorization scheme by Seattle DOT](https://sdotblog.seattle.gov/2017/08/28/seattle-sidewalk-survey-update/):

image

### LINKED ISSUE #2359: Can we add "narrow" as a tag for sidewalk obstacle
I previously mentioned this here https://github.com/ProjectSidewalk/SidewalkWebpage/issues/1979#issue-534410183 (now closed ticket) but I think we should add "narrow path" as a tag for sidewalk obstacle. Yes, I know this exists in surface problem but sometimes I think "sidewalk obstacle" is a more fitting label and I still want to tag it with narrow.

### PR DESCRIPTION
**Pull request by jacknussbaum**
Resolves #2339 
Resolves #2359 

Adds 'height difference' to surface problems, changes obstacles' 'large step' tag to 'height difference'. Adds 'narrow' tag to the Obstacles as well. 

New tags for obstacle:
image

New tag for Surface Problems:
image

### CODE REVIEW

File: public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js [hunk 1]
```diff
@@ -439,7 +439,7 @@ function ContextMenu (uiContextMenu) {
      * @param label     Current label being modified.
      */
     function setTags (label) {
-        var maxTags = 10;
+        var maxTags = 11;
```
Comment (jacknussbaum) [inline public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js:442]: I just incremented this by 1 for now... is there a better change you want me to implement here?

Comment (misaugstad) [reply public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js:442]: Nah that's probably fine for now. I don't have a good alternative yet.

**Comment by misaugstad** [comment]
Here are some updates to make before I finish reviewing:
* Change the "height difference" keyboard shortcut from `L` to `D` for both surface problems and obstacles (this works for the Spanish translation "desnivel" particularly well)
* Change the keyboard shortcut for "street vendor" from `D` to `U` to accommodate the above change
* Change the new tag from "narrow" to "narrow sidewalk" so that it matches the surface problem version
* Change the keyboard shortcut for the old "narrow sidewalk" tag to also be `W` so that it matches the new obstacle version
* Can you paste 4 screenshots when you're done: a Spanish version and an English version, from a US city and a Mexico city. It looks like you have a database dump for a city in Mexico, so I'll send you a small database dump from a US city as well.

**Comment by jacknussbaum** [comment]
Oh boy. Im not sure what I did with all those commits shown above..? I pulled from develop to update before I committed my changes, not sure why all these past commites are showing up here... I made the fixes you requested to all of the key shortcut letters and such.

**Comment by jacknussbaum** [comment]
image
Ever seen an error like this? Not sure what could be causing this, as I only changed the lettering since this PR was functioning..

**Comment by misaugstad** [comment]
>Oh boy. I'm not sure what I did with all those commits shown above..? I pulled from develop to update before I committed my changes, not sure why all these past commits are showing up here... I made the fixes you requested to all of the key shortcut letters and such.

Yeah I'm not totally sure what happened there... It concerns me a bit :grimacing: Can you try to checkout out the commit before things got messed up, force push it so it overwrites all the weird stuff that happened, merge develop into the branch, and then remake the new commit? I think it would look something like:
1. Check out the commit before things went haywire:
    ```
    git checkout 2339-Add-New-Tag-Types
    get checkout f4b3372376873b57c139be563359860205ab507e
    ```
1. Force push (I think this should work..?): `git push origin 2339-Add-New-Tag-Types --force`
1. Merge in develop: `git pull origin develop`
1. Manually remake those the edits in your "Changed key letters for new tags" commit (if you've force pushed to Github, you won't see them on Github anymore, so maybe just open up a tab with that commit so you can reference them easily). Then
    ```
    git commit -m "Changed key letters for new tags"
    git push origin 2339-Add-New-Tag-Types
    ```

You should probably also run the code after each of those steps to make sure that the bug you showed above doesn't show up again. And if it shows up at some point, you'll know where it was introduced.

And as for that bug... Yeah there is some very old code that used to let you add polygons using that method instead of single labels. We are planning on completely removing it soon. The issue for it is #956. If you still have issues with it after fixing the branch like above, maybe take a look at that thread because it might point you in the right direction to fix the issue.

**Comment by jacknussbaum** [comment]
Got the Columbus db set up. Make dev and the containers all started fine, can view the whole db in valentine, but am getting this error when accessing the page...
image
I altered all the lines in the docker-compose, the shell returns the error saying the database 'sidewalk-columbus' doesn't exist. I will continue to troubleshoot this though..

Good news: I fixed the polygons issue and have all the code working for spgg. Screenshots below. Thanks for your help with the git stuff... still learning all the ins and outs of version control apparently.

**Mexican City:**

In English:
image
image

In Spanish:
image
image

**Review by misaugstad — CHANGES_REQUESTED** [review]
For surface problems, the 'height difference' tag's keyboard shortcut isn't working. Otherwise, everything looks good!

**Comment by jacknussbaum** [comment]
Should be all fixed up now!

**Comment by jacknussbaum** [comment]
Updated the evolutions file to 99.sql to resolve the conflict when a new 98.sql was created elsewhere.

**Review by misaugstad — APPROVED** [review]
Looks good, thanks @jacknussbaum !!

**Pull request merged**